tests/bearlib/languages/documentation/documentation_extraction_testdata/default.c

```diff
@@ -33,7 +33,8 @@ int foobar(int x) {
 }
 
      /** line 1
-      *  line2 */
+      *  line2
+      */
```

Comment:
same as the case of above. But this file is used for `assemble()` test. Which adds extra space at the end like 
```
* line2  */
```
This is only happening in case of `C`